src/components/media-loader.js

```diff
@@ -190,14 +197,29 @@ AFRAME.registerComponent("media-loader", {
       } else if (contentType.startsWith("image/")) {
         this.el.removeAttribute("gltf-model-plus");
         this.el.removeAttribute("media-video");
-        this.el.addEventListener("image-loaded", this.onMediaLoaded, { once: true });
         this.el.removeAttribute("media-pager");
+        this.el.addEventListener(
+          "image-loaded",
+          () => {
+            const mayChangeScene = !!window.APP.getPermissions().update_hub;
+
+            if (isHubsRoomUrl(src) || (isHubsSceneUrl(src) && mayChangeScene)) {
+              this.el.setAttribute("hover-menu__hubs-item", {
```

Comment:
technically if this `media-loader` changes type this will stick around, this is also true of the video menus though. Probably not for this PR but I vote we just stop pretending media-loaders can change type and just throw an error if it tries to. That allows us to delete a lot of the ugly `removeAttribute` code littered about in this `update` function.